VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionPostsThreadFragment.java

```diff
@@ -74,9 +85,22 @@ public void onCreate(Bundle savedInstanceState) {
         EventBus.getDefault().register(this);
 
         if (discussionTopic == null) {
-            // Either we are coming from a deep link or courseware inline discussion
-            discussionTopic = new DiscussionTopic();
-            discussionTopic.setIdentifier(getArguments().getString(Router.EXTRA_DISCUSSION_TOPIC_ID));
+            // Either we are coming from a deep link or courseware's inline discussion
+            String topicId = getArguments().getString(Router.EXTRA_DISCUSSION_TOPIC_ID);
+            GetSpecificCourseTopicsTask getTopicsTask = new GetSpecificCourseTopicsTask(getContext(),
+                    courseData.getCourse().getId(), Collections.singletonList(topicId)) {
+                @Override
+                protected void onSuccess(CourseTopics courseTopics) throws Exception {
+                    super.onSuccess(courseTopics);
```

Comment:
The super implementation doesn't do anything, so this invocation can be removed.